bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
```

Comment:
timestamp returns a fixed length string. You can always convert to int before writing it to the database. I am just concerned of problems that may arise if we start getting timestamps from different sources in different parts of the code